### DISCUSSION TITLE
Rotating GlobeView by fixed angle

### DISCUSSION THREAD

**Discussion by Mstoltz17** [visgl/deck.gl#9116] [Q&A]
I am creating a satellite simulation that requires the earth to be rotated (only worried about rotation about the polar axis i.e. longitude only), controlled by a time slider.
I have a GlobeView layer with a number of GeoJsonLayers to create the earth model. When the simulation time changes I calculate how much the earth should be rotated and perform the rotation by shifting the longitude of every point in the geoJsons by that value. It works but is very slow considering the simulation time updates ~20 times/s. 
Is there a faster way to achieve the same effect?
Thanks in advance!

**Comment by Pessimistress** [comment]
You should be using [Camera transitions](https://deck.gl/docs/developer-guide/animations-and-transitions#camera-transitions)
